osu.Game/Beatmaps/BeatmapConverter.cs

```diff
@@ -8,12 +8,36 @@
 
 namespace osu.Game.Beatmaps
 {
+    public interface ITestableBeatmapConverter

```

Comment:
Can probably just call this `IBeatmapConverter`, doesn't have anything specific to testing here.

Also move to its own file please.

Reply:
I really don't want to expose this as an `IBeatmapConverter`, if not for the fact that it doesn't give you a resulting `Beatmap`, just an event to consume.